src/backend/columnar/columnar_metadata.c

```diff
@@ -71,6 +71,7 @@ static void GetHighestUsedAddressAndId(uint64 storageId,
 									   uint64 *highestUsedAddress,
 									   uint64 *highestUsedId);
 static List * ReadDataFileStripeList(uint64 storageId, Snapshot snapshot);
+static StripeMetadata * BuildStripeMetadata(Datum *datumArray);
```

Comment:
Is this related to indexes at all, or is it just cleanup?

Reply:
I'm using that function in `FindStripeByRowNumber` function (which returns the stripe metadata for the stripe that has the row with `rowNumber`). So it's actually related with indexes.

Reply:
update: later, we merged that function via #4958